store/sqlstore/upgrade.go

```diff
@@ -170,17 +169,17 @@ func UpgradeDatabase(sqlStore SqlStore, currentModelVersionString string) error
 
 func saveSchemaVersion(sqlStore SqlStore, version string) {
 	if err := sqlStore.System().SaveOrUpdate(&model.System{Name: "Version", Value: version}); err != nil {
-		mlog.Critical(err.Error())
+		mlog.Critical("critical", mlog.Err(err))
```

Comment:
I'm just questioning if this change adds value... what is the thinking here?

Reply:
oh yes Sprintf is not used here no need to change reverting.

Reply:
reverted